autoload/db_ui/schemas.vim

```diff
@@ -34,9 +34,16 @@ let s:postgres_list_schema_query = "
     \   and pg_catalog.has_schema_privilege(current_user, nspname, 'USAGE')
     \ order by nspname"
 
-let s:postgres_tables_and_views = "
-      \ SELECT table_schema, table_name FROM information_schema.tables UNION ALL
-      \ select schemaname, matviewname from pg_matviews;"
+if !empty(g:db_ui_use_redshift)
```

Comment:
Is the only difference that redshift does not support views ? If yes, we should change the variable to be something like `g:db_ui_use_postgres_views` which would default to true, and then if you set it to false, it will not UNION the views.

Reply:
The only difference is indeed that redshift does not support views in the same way that postgres does. Seeing the tables inside the schemas is the only thing that this changes.

![dadbod_example](https://github.com/kristijanhusak/vim-dadbod-ui/assets/59935328/dc881f3a-6c2a-4e27-9ef8-554e1fcc3ed7)


The table helpers modification wasn't necessary and was just left over from testing so I removed it. 

Where do you think would be best to define the `g:db_ui_use_postgres_views` variable so that it can be overwritten by a user?